angularjs/angular-cookies.d.ts

```diff
@@ -11,23 +11,23 @@ declare module "angular-cookies" {
     export = _;
 }
 
-///////////////////////////////////////////////////////////////////////////////
-// ngCookies module (angular-cookies.js)
-///////////////////////////////////////////////////////////////////////////////
+/**
+ * ngCookies module (angular-cookies.js)
+**/

```

Comment:
There's an extra `*` on this line, and below on L22 and L30. It's another nitpick but I point it out just in case this causes some TS editor plugin to fail to read this as a JSDoc comment (albeit that would be a poor plugin implementation, but still).